ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-provisioned-dns/private/cucushift-installer-rehearse-gcp-ipi-user-provisioned-dns-private-workflow.yaml

```diff
@@ -1,6 +1,8 @@
 workflow:
   as: cucushift-installer-rehearse-gcp-ipi-user-provisioned-dns-private
   steps: 
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
```

Comment:
for private cluster testing, I guess we can use a random domain here since there's no limitation on this, just like "custom-dns.com", wdyt?

Reply:
Just updated accordingly, thanks!